pagopa/pn-frontend-e2e
# PN-10031-e-2-e-destinatario-inserimento-indirizzo-email-come-altro-recapito

### Issue Description
Some PF's tests are failing due to some bad background API integration. In this case the **PN-9318-I40** is failing because it doesn't remove the contacts once the test is finished and this will cause a failure after re-run the test multiple times.

### Changes Made:
- adjust some feature steps by lowering `waitTime` and fixing correct `webelements`
- add new background API related to `contacts`
- add `@After` into `Hooks` so each `@recapitiPF` revokes all contacts at the end of the test in order to clear the data preparation for the next re-runs
- add new models/rest classes related to `contacts`

### How to Test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_emailAggiuntivaPF` and verify if the test works fine.
Also a run into AWS CodeBuild is necessary in order to test everything.
